static/app/utils/useKeyPress.tsx

```diff
@@ -1,33 +1,39 @@
-import {useEffect, useState} from 'react';
+import {RefObject, useEffect, useState} from 'react';
 
 /**
  * Hook to detect when a specific key is being pressed
  */
-function useKeyPress(targetKey: React.KeyboardEvent['key']) {
+const useKeyPress = (targetKey: string, ref?: RefObject<HTMLInputElement>) => {
   const [keyPressed, setKeyPressed] = useState(false);
 
-  useEffect(() => {
-    function downHandler({key}: KeyboardEvent) {
-      if (key === targetKey) {
-        setKeyPressed(true);
-      }
+  const downHandler = ({key}: KeyboardEvent) => {
+    if (key === targetKey) {
+      setKeyPressed(true);
     }
+  };
```

Comment:
Why did these move out? Now they're redefined on every render (which still magically works because we're not removing and adding the listeners on *every* render)